src/stt_data_with_llm/catalog_parser.py

```diff
@@ -1,2 +1,73 @@
-def parse_catalog(catalog):
-    pass
\ No newline at end of file
+import logging
+
+import pandas as pd
+
+# Configure logging
+logging.basicConfig(level=logging.INFO)
```

Comment:
if you are going to show the logging in terminal only how are you going to analysis the log later. I would save the log in a log file.

Reply:
@kaldan007, done with the changes.